<ui version="4.0" >
 <class>CineCastUiClass</class>
 <widget class="QWidget" name="CineCastUiClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CineCastUi</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLabel" name="label_logoBig" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="cinecastui.qrc" >:/CineCastUi/Resources/logobig.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_LOGO" >
       <property name="font" >
        <font>
         <family>Book Antiqua</family>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>  CineCast</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="font" >
      <font>
       <family>Book Antiqua</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="styleSheet" >
      <string notr="true" >QTabWidget#Tab1>QTabBar::tab:first
{
    border-image: url(:/NULLbackground.png);
}
QTabWidget#Tab1>QTabBar::tab:first:hover
{
    border-image: url(:/NULLbackground.png);
}
QTabWidget#Tab1>QTabBar::tab:first:pressed
{
    border-image: url(:/NULLbackground.png);
}
QTabWidget#Tab1>QTabBar::tab:last
{
    border-image: url(:/NULLbackground.png);
}
QTabWidget#Tab1>QTabBar::tab:last:hover
{
    border-image: url(:/NULLbackground.png);
}
QTabWidget#Tab1>QTabBar::tab:last:pressed
{
    border-image: url(:NULLbackground.png);
}</string>
     </property>
     <property name="tabShape" >
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex" >
      <number>3</number>
     </property>
     <property name="iconSize" >
      <size>
       <width>20</width>
       <height>35</height>
      </size>
     </property>
     <property name="usesScrollButtons" >
      <bool>true</bool>
     </property>
     <property name="tabsClosable" stdset="0" >
      <bool>false</bool>
     </property>
     <property name="movable" stdset="0" >
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab_1" >
      <attribute name="title" >
       <string>Status</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QMdiArea" name="mdiArea_statue" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Content</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <widget class="QMdiArea" name="mdiArea_ContentManagement" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Manager</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QMdiArea" name="mdiArea_Maintain" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4" >
      <attribute name="title" >
       <string>Setup</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <item>
        <widget class="QMdiArea" name="mdiArea_Setup" >
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="cinecastui.qrc" />
 </resources>
 <connections/>
</ui>
